src/frontend/screens/WineManager/index.tsx

```diff
@@ -110,6 +114,43 @@ export default function WineManager(): JSX.Element | null {
     }
   }, [repository])
 
+  let wineVersionExplanation = <></>
```

Comment:
The `useMemo` hook should be used for values like this, e.g.
```ts
const wineVersionExplanation = useMemo(() => { /* switch case in here, returning the appropriate value */ }, [repository.type])
```

That'll prevent the value from being re-calculated on every re-render